src/sync/DAL/typeorm/syncRepository.ts

```diff
@@ -29,4 +29,12 @@ export class SyncRepository extends Repository<DbSync> implements ISyncRepositor
     }
     return syncEntity;
   }
+
+  public async findFullSyncByLayerAndGeometry(layerId: number, geometryType: GeometryType): Promise<Sync | undefined> {
+    const fullSyncEntity = await this.findOne({ where: { layerId, geometryType, isFull: true } });
+    if (!fullSyncEntity) {
+      return undefined;
+    }
+    return fullSyncEntity;
+  }
```

Comment:
change it so isFull is also a parameter of the function.

Reply:
changed to `findSyncs` which is dynamic to any filter